pkgs/applications/blockchains/alfis/default.nix

```diff
@@ -36,6 +41,9 @@ rustPlatform.buildRustPackage rec {
     ++ lib.optional (withGui && stdenv.isLinux) webkitgtk
     ++ lib.optionals (withGui && stdenv.isDarwin) [ Cocoa WebKit ];
 
+  NIX_CFLAGS_COMPILE = lib.optionalString (stdenv.isDarwin && stdenv.isAarch64)
+    "-DOBJC_OLD_DISPATCH_PROTOTYPES=1";
```

Comment:
How would this even work? In my testing (for another project) `OBJC_OLD_DISPATCH_PROTOTYPES` was ignored by the compiler for arm64 targets (it assumed it's 0). Therefore that project had to cast all calls.

The PR description you provide also suggests it gets ignored on arm64; however, here you explicitly set it for arm64.

Reply:
Oh, I didn't mean to put that in the commit. Sorry!